valueformatting_test.go

```diff
@@ -551,11 +551,11 @@ func TestPrintRow(t *testing.T) {
 	want :=
 		"----------------------------------------\n" +
 			"r1\n" +
-			"  f1:c1                                    @ 1969/12/31-16:00:00.000000\n" +
```

Comment:
I'm also curious what the difference was. If it's just about the epoch, what happens when we use a non-zero timestamp?

Reply:
The difference was that my Mac printed out the epoch string as shown above, but Kokoro was formatting it like `@ 1970/01/01-00:00:00.000000` for the exact same inputs.

Reply:
[facepalm] I think it's a time zone issue. Regardless, I think it would be easier to change the tests so that we're not checking for an exact timestamp string.

Reply:
(Thanks to @igorbernstein2 for catching this)